src/shared/components/event-management/components/task-conflict/task-conflict.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component, Input } from '@angular/core';
+import { Task } from '../../../../domain/work-allocation/Task';
+
+@Component({
+  selector: 'app-task-conflict',
+  templateUrl: './task-conflict.component.html'
+})
+export class TaskConflictComponent {
+  @Input() public task: Task = null;
+  @Input() public caseId = '1620409659381330';
```

Comment:
please remove hard code 1620409659381330

Reply:
Thank you @udaydenduluri33, I have now removed the hard-coded value.